test/e2e/dra/test-driver/README.md

```diff
@@ -48,9 +48,14 @@ kubelet<->dynamic resource allocation plugin interaction.
 
 ### `local-up-cluster.sh`
 
+etcd is required to be in your PATH, install it by calling
+```console
+./hack/install_etcd.sh
+```
+
 To try out the feature, build Kubernetes, then in one console run:
 ```console
-FEATURE_GATES=DynamicResourceAllocation=true KUBELET_RESOLV_CONF="/etc/resolv-9999.conf" DNS_ADDON="coredns" CGROUP_DRIVER=systemd CONTAINER_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock LOG_LEVEL=6 ENABLE_CSI_SNAPSHOTTER=false API_SECURE_PORT=6444 ALLOW_PRIVILEGED=1 PATH=/nvme/gopath/src/k8s.io/kubernetes/third_party/etcd:$PATH ./hack/local-up-cluster.sh -O
+FEATURE_GATES=DynamicResourceAllocation=true KUBELET_RESOLV_CONF="/var/run/systemd/resolve/resolv.conf" CGROUP_DRIVER=systemd CONTAINER_RUNTIME_ENDPOINT=unix:///var/run/crio/crio.sock LOG_LEVEL=6 API_SECURE_PORT=6444 ALLOW_PRIVILEGED=1 PATH="$(pwd)/third_party/etcd:$PATH" ./hack/local-up-cluster.sh -O
```

Comment:
I dont' have /var/run/systemd/resolve directory on my system, so this wouldn't work.
I'm not using KUBELET_RESOLV_CONF either as my /etc/resolv.conf points to the real name servers.

Reply:
I don't remember, but it could be the case. Not sure about Fedora either. I'm using Ubuntu 18.04 LTS.

Reply:
Fedora 36, Ubuntu 20.04, Ubuntu 22.04 seem to have that by default.

Reply:
I'd suggest to leave only env variables that are mandatory for DRA and emphasise this somehow. We can give a real example with KUBELET_RESOLV_CONF and other non-mandatory options though.

Reply:
I ended up simplifying the invocation in https://github.com/pohly/kubernetes/pull/20 before I noticed this PR.